### PR TITLE
Updates for 0.11.1

### PR DESCRIPTION
**Pull request by paf31**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,13 +2,14 @@
   "private": true,
   "scripts": {
     "clean": "rimraf output && rimraf .pulp-cache",
-    "build": "pulp build --censor-lib --strict",
+    "build": "jshint src && jscs src && pulp build --censor-lib --strict",
```
Comment (garyb) [inline package.json]: There's no JS in this library? Plus eslint has replaced both of these tools now.

File: .travis.yml [hunk 2]
```diff
@@ -1,17 +1,21 @@
...
+  - tar -xvf $HOME/purescript.tar.gz -C $HOME/
+  - chmod a+x $HOME/purescript
+  - npm install -g bower
   - npm install
-  - bower install --production
+  - bower install
 script:
   - npm run -s build
-  - bower install
-  - npm -s test
```
Comment (garyb) [inline .travis.yml]: Lost the script to run the tests here...

File: .travis.yml [hunk 3]
```diff
@@ -1,17 +1,21 @@
...
-node_js: 6
+node_js: stable
+env:
+  - PATH=$HOME/purescript:$PATH
 install:
-  - npm install bower -g
+  - TAG=$(wget -q -O - https://github.com/purescript/purescript/releases/latest --server-response --max-redirect 0 2>&1 | sed -n -e 's/.*Location:.*tag\///p')
+  - wget -O $HOME/purescript.tar.gz https://github.com/purescript/purescript/releases/download/$TAG/linux64.tar.gz
+  - tar -xvf $HOME/purescript.tar.gz -C $HOME/
+  - chmod a+x $HOME/purescript
```
Comment (garyb) [inline .travis.yml]: Do we want to do this in the contrib libraries?

File: .travis.yml (hunk 2 above)
Comment (paf31) [reply .travis.yml]: Yeah, I'm not sure what to do with Travis in general here, until there is a Pulp update.

File: package.json (hunk 1 above)
Comment (paf31) [reply package.json]: I just copy-pasted these to be honest.

File: .travis.yml (hunk 3 above)
Comment (paf31) [reply .travis.yml]: What do you mean?

Comment (garyb) [reply .travis.yml]: We only used to use this on the core libraries, and relied on a specific version of the compiler via `npm` for the contrib etc. libraries.

Comment (paf31) [reply .travis.yml]: I guess we should wait for Pulp 11 then?

Comment (paf31) [reply .travis.yml]: I made some contrib updates earlier in the week which had a lot of these same problems actually. I need to also go back and fix those up.

**Pull request merged**